app/jinja_filters.py

```diff
@@ -66,6 +66,12 @@ def format_multilined_string(context, value):
     return result
 
 
+@blueprint.app_template_filter()
+def get_current_date(date_format='%d %B %Y'):
+    now = datetime.utcnow().replace(tzinfo=tz.gettz('UTC')).astimezone(tz.gettz('Europe/London')).strftime(date_format)
```

Comment:
Could we abstract this bit out into a separate method (`as_london_tz()`) and also call it from `format_datetime()`?

`.replace(tzinfo=tz.gettz('UTC')).astimezone(tz.gettz('Europe/London'))`

Reply:
Wouldn't `datetime.now().astimezone(tz.gettz('Europe/London'))` be easier. We don't need to get UTC time and convert it. We just need local London time.